Starcounter.Async/Internal/StarcounterSessionSynchronizationContext.cs

```diff
@@ -25,13 +23,14 @@ public StarcounterSessionSynchronizationContext()
 #pragma warning disable CS0618 // Type or member is obsolete - Session.ScheduleTask
         public override void Send(SendOrPostCallback d, object state)
         {
-            Session.ScheduleTask(SessionId,
-                (Session session, string sessionId) => {
+            Session.RunTask(SessionId,
+                (Session session, string sessionId) =>
+                {
                     if (session != null)
                     {
                         d(state);
                     }
-                }, waitForCompletion: true);
```

Comment:
I see you removed `waitForCompletion` argument. It doesn't exist in `Session.RunTask` or its not needed anymore?

Reply:
Yeah, it doesn't exists in `Session.RunTask`.